e2e/flags.go

```diff
@@ -91,6 +91,16 @@ var KCC = flag.Bool("kcc", false,
 var GceNode = flag.Bool("gcenode", false,
 	"If true, run test with 'gcenode' auth type.")
 
+// GitHubApp enables running the e2e tests for 'githubapp' auth type
+var GitHubApp = flag.Bool("githubapp", false,
+	"If true, run test with 'githubapp' auth type.")
+
+// GitHubAppConfigFile specifies a file path to read GitHub App config from.
+// If unset, defaults to fetching the GitHub App config from Secret Manager.
+var GitHubAppConfigFile = flag.String("githubapp-config-file",
+	util.EnvString("E2E_GITHUBAPP_CONFIG_FILE", ""),
```

Comment:
We can probably create a secret in the oss-prow-build cluster. That will enable the github e2e test in the presubmit job.

Reply:
That may work, but regardless of how the secret is provisioned I don't think we should run this test as part of the presubmit jobs. The ok-to-test label is sticky, so if we set this up to run as a presubmit then essentially any contributor can read the secret. At that point we may as well store the secret as plaintext in the repo, which we shouldn't do for obvious reasons.

Reply:
Make sense. Then we'll rely on postsubmit CIs to discover any potential break to the github app authentication.